File: v2/jaxrs/src/main/java/io/swagger/model/Newsletters.java
package io.swagger.model;

import io.swagger.model.Newsletter;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Newsletters  {
  
  private List<Newsletter> newletters = new ArrayList<Newsletter>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("newletters")
  public List<Newsletter> getNewletters() {
    return newletters;
  }
  public void setNewletters(List<Newsletter> newletters) {
    this.newletters = newletters;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Newsletters {\n");
    
    sb.append("  newletters: ").append(newletters).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Newsletters.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Newsletters {
    

    
    public List<Newsletter> Newletters { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Newsletters {\n");
      
      sb.Append("  Newletters: ").Append(Newletters).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}